tools/sigclient/pkg/query/query.go

```diff
@@ -921,6 +921,9 @@ func RunQueryFromFileAndOutputResponseTimes(dest string, filepath string, queryR
 }
 
 var skipIndexes = map[int]bool{
+	// Working now
+	4: true, // Not working Pipe QL
```

Comment:
Why do you say `Working now` but list it in the indexes to skip?

Reply:
addressed in the latest commit. It is not working